tests_integration/test_data/projects/snowpark_v2/requirements.txt

```diff
@@ -0,0 +1 @@
+snowflake-snowpark-python
syrupy

```

Comment:
No idea why github shows this like this while the file looks fine

Reply:
https://stackoverflow.com/questions/5843495/what-does-the-m-character-mean-in-vim 😄